straxen/rundb.py

```diff
@@ -105,8 +105,9 @@ def __init__(self,
         self.collection = self.client[mongo_dbname][mongo_collname]
 
         self.backends = [
-            strax.S3Backend(**s3_kwargs),
-            strax.FileSytemBackend(),
+            #strax.S3Backend(**s3_kwargs),
```

Comment:
Why not removing those lines?

Reply:
These lines correspond to backends that still exist, so you have the choice between removing the comments, which will make you have 3 backends in all, or leaving it like that, which will make it go faster.

Reply:
Okay I do not get it why do we have to comment out those existing backends? Either they exist and we use them then we should not comment them out, or they do not exist and we do not use them, in this case we can remove those lines.

Reply:
You are right, I will remove the comments